README.md

```diff
@@ -102,7 +102,11 @@ Install with [Scoop](https://scoop.sh):
 ```powershell
 scoop install lsd
 ```
-
+### On Android (via Termux)
+```sh
+pkg install lsd
+```
+`lsd` require Nerd Font to showing icons properly. [Setup Nerd Font in Termux](https://github.com/Peltoche/lsd/issues/423)
```

Comment:
```suggestion
[Setup Nerd Fonts in Termux](https://github.com/Peltoche/lsd/issues/423)
```

What do you think? Since we have already mentioned the dependency of Nerd Fonts above in the readme.

Reply:
Oh, okay. Thanks